metadata-ingestion/src/datahub/ingestion/source/usage/usage_common.py

```diff
@@ -62,11 +62,18 @@ def trim_query(self, query: str, budget_per_query: int) -> str:
                 )
         return trimmed_query
 
+    def indent_query(self, query: str, indent) -> str:
+        if indent:
+            return sqlparse.format(query, keyword_case="upper", reindent_aligned=True)
```

Comment:
Wouldn't this raise exceptions if the query is not properly formed? Have you tried testing this with trimmed queries?

Reply:
It does not raise exception. If the query is not properly formed, it returns the query as is.